<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TerminalEditDialog</class>
 <widget class="QDialog" name="TerminalEditDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>270</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Terminal Info</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="terminal_edit_grid" rowstretch="0,0,0" columnstretch="0,1,1">
     <item row="1" column="1">
      <widget class="QLineEdit" name="unfinished_teleport_edit"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="terminal_id_label">
       <property name="text">
        <string>Terminal ID:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="unfinished_teleport_label">
       <property name="text">
        <string>Unfinished Teleport:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="finished_teleport_label">
       <property name="text">
        <string>Finished Teleport:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="finished_teleport_edit"/>
     </item>
     <item row="1" column="2">
      <widget class="QComboBox" name="unfinished_teleport_type"/>
     </item>
     <item row="2" column="2">
      <widget class="QComboBox" name="finished_teleport_type"/>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QLineEdit" name="terminal_id_edit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="dialog_button_box">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
